### ISSUE TITLE
CC Lib: library loaded in panel doesn't stay in sync moving between DS and classic PS

### ISSUE THREAD

**Issue by chadrolfs** [adobe-photoshop/spaces-design#2360]
load a library in DS, switch to classic and verify it is the same library, if it is either switch to a different library in classic and then switch to DS, or switch to DS change to a different library and switch back to classic. 

Result: you end up having a different library loaded in DS vs. classic PS.

**Comment by shaoshing** [comment]
I agree that this is confusing and would be nice if we can keep sync of the selected library between DS and PS. However, it seems difficult in the current, stage given the fact that CEP does not share its state with with DS (or even Photoshop), and it is unloaded when switch to DS.

**Comment by iwehrman** [comment]
@shaoshing Can you stuff this into JIRA somehow?
